### ISSUE TITLE
[Feature request] Read embedded synced lyrics

### ISSUE THREAD

**Issue by BobbyESP** [Kyant0/Tag#2]
It would be nice if we could read the embedded synced lyrics of a song. ID3 supports it.

**Comment by Kyant0** [comment]
We already support it, doesn't this work for you?

**Comment by Kyant0** [comment]
You can use `Metadata.getLyrics(path)` and  it returns lyrics string, You should manually parse it to the format you want.

**Comment by BobbyESP** [comment]
By using getLyrics I just get the plain ones. There should be another ID3 field for synced ones

**Comment by BobbyESP** [comment]
I think it is SYTL for synced and USTYL for unsynced

**Comment by Kyant0** [comment]
Not exactly, USLT is for unsynced, but it seems to be more common to put the synced to the unsynced chunk?
However, we should consider both of them.

**Comment by BobbyESP** [comment]
Any update about the ability to edit the metadata?
